zsh-nvm.plugin.zsh

```diff
@@ -93,6 +93,7 @@ _zsh_nvm_lazy_load() {
 
   # Add nvm
   global_binaries+=('nvm')
+  global_binaries+=$NVM_LAZY_LOAD_EXTRA_COMMANDS
```

Comment:
You are appending `$NVM_LAZY_LOAD_EXTRA_COMMANDS` as a space separated string here.

You need to wrap with parenthesis to merge the arrays:


```suggestion
  global_binaries+=($NVM_LAZY_LOAD_EXTRA_COMMANDS)
```

Reply:
te alright I'll update it!